src/transformers/models/poolformer/configuration_poolformer.py

```diff
@@ -0,0 +1,131 @@
+    architecture. Instantiating a configuration with the defaults will yield a similar configuration to that of
+    the PoolFormer `sail/poolformer_s12 <https://huggingface.co/sail/poolformer_s12>`__ architecture.
+
+    Configuration objects inherit from  :class:`~transformers.PretrainedConfig` and can be used
+    to control the model outputs. Read the documentation from  :class:`~transformers.PretrainedConfig`
+    for more information.
+
+
+    Args:
+        num_channels (:obj:`int`, optional, defaults to 3):
+            The number of channels in the input image.
+        patch_size (:obj:`int`, optional, defaults to 16):
+            The size of the input patch.
+        stride (:obj:`int`, optional, defaults to 16):
+            The stride of the input patch.
+        pool_size (:obj:`int`, optional, defaults to 3):
+            The size of the pooling window.
+        mlp_ratio (:obj:`float`, optional, defaults to 4.0):
+            The ratio of the number of channels in the output of the MLP to the number of channels in the input.
+        depths (:obj:`list`, optional, defaults to [2, 2, 6, 2]):
+            The depth of each encoder block.
+        hidden_sizes (:obj:`list`, optional, defaults to [64, 128, 320, 512]):
+            The hidden sizes of each encoder block.
+        patch_sizes (:obj:`list`, optional, defaults to [7, 3, 3, 3]):
+            The size of the input patch for each encoder block.
+        strides (:obj:`list`, optional, defaults to [4, 2, 2, 2]):
+            The stride of the input patch for each encoder block.
+        padding (:obj:`list`, optional, defaults to [2, 1, 1, 1]):
+            The padding of the input patch for each encoder block.
+        num_encoder_blocks (:obj:`int`, optional, defaults to 4):
+            The number of encoder blocks.
+        drop_path_rate (:obj:`float`, optional, defaults to 0.0):
+            The dropout rate for the dropout layers.
+        hidden_act (:obj:`str`, optional, defaults to "gelu"):
+            The activation function for the hidden layers.
+        use_layer_scale (:obj:`bool`, optional, defaults to True):
+            Whether to use layer scale.
+        layer_scale_init_value (:obj:`float`, optional, defaults to 1e-5):
+            The initial value for the layer scale.
+        initializer_range (:obj:`float`, optional, defaults to 0.02):
+            The initializer range for the weights.
+        use_cache (:obj:`bool`, optional, defaults to True):
+            Whether to use cache.
+        Example::
+
+        >>> from transformers import PoolFormerModel, PoolFormerConfig
+
+        >>> # Initializing a PoolFormer sail/poolformer_s12 style configuration
+        >>> configuration = PoolFormerConfig()
+
+        >>> # Initializing a model from the sail/poolformer_s12 style configuration
+        >>> model = PoolFormerModel(configuration)
+
+        >>> # Accessing the model configuration
+        >>> configuration = model.config
```

Comment:
```suggestion
        
This code example needs to be shifted to the left, otherwise it won't properly render. See configuration_vit.py as an example.

        >>> from transformers import PoolFormerModel, PoolFormerConfig

        >>> # Initializing a PoolFormer sail/poolformer_s12 style configuration
        >>> configuration = PoolFormerConfig()

        >>> # Initializing a model from the sail/poolformer_s12 style configuration
        >>> model = PoolFormerModel(configuration)

        >>> # Accessing the model configuration
        >>> configuration = model.config
```